src/modules/SessionReplay.ts

```diff
@@ -1,5 +1,24 @@
-import { NativeSessionReplay } from '../native/NativeSessionReplay';
+import { NativeSessionReplay, NativeEvents, emitter } from '../native/NativeSessionReplay';
+import type { LaunchType } from '../utils/Enums';
+export interface networkLog {
+  url: string;
+  duration: number;
+  statusCode: number;
+}
 
+export interface sessionData {
+  appVersion: string;
+  OS: string;
+  device: string;
+  sessionDurationInSeconds: number;
+  networkLogs: Array<networkLog>;
+  launchType: LaunchType;
+  hasLinkToAppReview: boolean;
+  launchDuration: number;
+  bugsCount?: number;
+  fatalCrashCount?: number;
+  oomCrashCount?: number;
+}
```

Comment:
Let's rename these to `NetworkLog` and `SessionMetadata` to follow the naming convention for interfaces. Let's also move them to the `models` directory instead.

```suggestion
export interface NetworkLog {
  url: string;
  duration: number;
  statusCode: number;
}

export interface SessionMetadata {
  appVersion: string;
  OS: string;
  device: string;
  sessionDurationInSeconds: number;
  networkLogs: NetworkLog[];
  launchType: LaunchType;
  hasLinkToAppReview: boolean;
  launchDuration: number;
  bugsCount?: number;
  fatalCrashCount?: number;
  oomCrashCount?: number;
}
```